source/traefik_proxy_test.go

```diff
@@ -19,9 +19,10 @@ package source
 import (
 	"context"
 	"encoding/json"
-	"k8s.io/apimachinery/pkg/runtime/schema"
 	"testing"
 
+	"k8s.io/apimachinery/pkg/runtime/schema"
```

Comment:
Please avoid unrelated changes. Consider filing a separate PR for linting instead